README.md

```diff
@@ -1,40 +1,37 @@
-# Signal Android 
+# Signal Android Websockets-Only
 
 Signal is a messaging app for simple private communication with friends.
 
 Signal uses your phone's data connection (WiFi/3G/4G) to communicate securely, optionally supports plain SMS/MMS to function as a unified messenger, and can also encrypt the stored messages on your phone.
 
-Currently available on the Play store.
```

Comment:
We could add the F-Droid repo from @xmikos (https://eutopia.cz/experimental/fdroid/repo/) and mention this as an alternative to self-building. Is there some way to make the app hash comparable so that self-compiled ones and the ones from @xmikos match? To be sure it's not modified in a mallicous way?